apps/frontend/src/app/details/details.page.html

```diff
@@ -0,0 +1,55 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-menu-button></ion-menu-button>
+    </ion-buttons>
+    <ion-title>Messages</ion-title>
+  </ion-toolbar>
+</ion-header>
+<ion-content>
+  <ion-button id="open-modal" expand="full">Create message</ion-button>
+  <ion-loading *ngIf="messages===[]"></ion-loading>
```

Comment:
looks strange, therefor use: !messages.length

or put ngif to ion-list:
<ion-list ngif="messages.length; else loading">

<ng-template #loading>
  <ion-loading *ngIf="messages===[]"></ion-loading>
</ng-template>